modules/mocked_authentication/spec/lib/mocked_credential/service_spec.rb

```diff
@@ -0,0 +1,126 @@
+                'pbCPzPmpvaG4uYnJhbWxleUBhZGhvY3RlYW0udXMiLCJlbHqZbF92ZXJpZmllZCI6dHJ1ZSwiZ2l2ZW5fb' \
+                'mFtZSI6IkpvaG4iLCJmYW1pb1bRfbmFtZSI6IkJyYW1sZXkiLCJiaXJ0aGRhdGUiOiIxOTg5LTAzLTI4Ii' \
+                'wic29jaWFsX3NlY3VyaXR5X251bWJlciI6IjA1Ni03Ni03MTQ5IiwidmVyaWZpZWRfYXQiOjE2MzU0NjUy' \
+                'ODYsImFjciI6Imh0dHA6Ly9pZG1hbmFnZW1lbnQuZ292L25zL2Fzc3VyYW5jZS9pYWwvMiIsIm5vbmNlIj' \
+                'oiYjIwNjc1ZjZjYmYwYWQ5M2YyNGEwMzE3YWU3Njk5OTQiLCJhdWQiOiJ1cm46Z292OmdzYTpvcGVuaWRj' \
+                'b25uZWN0LnByb2ZpbGVzOnNwOnNzbzp2YTpkZXZfc2lnbmluIiwianRpIjoicjA1aWJSenNXSjVrRnloM1' \
+                'ZuVlYtZyIsImF0X2hhc2giOiJsX0dnQmxPc2dkd0tKemc2SEFDYlJBIiwiY19oYXNoIjoiY1otX2F3OERj' \
+                'SUJGTEVpTE9QZVNFUSIsImV4cCI6MTY0NTY0MTY0NSwiaWF0IjoxNjQ1NjQwNzQ1LCJuYmYiOjE2NDU2ND' \
+                'A3NDV9.S3-8X9clNcwlH2RU5sNoYf9HXpcgVK9UGUJumhL2-3rvznrt6yGvkXvY4FuUzWEcI22muxUjbbs' \
+                'ZHjCfDImZ869NTWsI-DKohSNmNnyOom29LJRymJTn3htI5MNmpGwbmNWNuK5HgerPZblL44N1a_rqfTF4l' \
+                'ANQX0u52iIVDarcexpX0e9yS1rEPqi3PDdcwN_1tUYox4us9rgzRZaaoa4iTlFfovY7dfgo_ewqv2EDh7J' \
+                'SfJJQhFhyabkJ9HgNkkc4m0SHqztterZ6lHgIoiJdQot6wsL9pQTYzFzgHV830ltpjVUcLG5vMXw4Kqs3B' \
+                'N9tdSToHdB50Paxyfq9kg'
+    }
+  end
+  let(:user_info) do
+    OpenStruct.new({
+                     sub: user_uuid,
+                     iss: 'https://idp.int.identitysandbox.gov/',
+                     email: email,
+                     email_verified: true,
+                     given_name: first_name,
+                     family_name: last_name,
+                     address: address,
+                     birthdate: birth_date,
+                     social_security_number: ssn,
+                     verified_at: 1_635_465_286
+                   })
+  end
+  let(:first_name) { 'Bob' }
+  let(:last_name) { 'User' }
+  let(:birth_date) { '1993-01-01' }
+  let(:ssn) { '999-11-9999' }
+  let(:address) do
+    {
+      formatted: formatted_address,
+      street_address: street,
+      postal_code: postal_code,
+      region: region,
+      locality: locality
+    }
+  end
+  let(:formatted_address) { "#{street}\n#{locality}, #{region} #{postal_code}" }
+  let(:street) { "1 Microsoft Way\nApt 3" }
+  let(:postal_code) { '11364' }
+  let(:region) { 'NY' }
+  let(:locality) { 'Bayside' }
+  let(:multifactor) { true }
+  let(:email) { 'user@test.com' }
+  let(:user_uuid) { '12345678-0990-10a1-f038-2839ab281f90' }
+
+  describe '#token' do
+    context 'when the request is successful' do
+      it 'returns an access token' do
+        subject.token(code)
+      end
+    end
+  end
+
+  describe '#user_info' do
+    context 'when the request is successful' do
+      let(:mocked_authorization_credential_information) { 'mocked_authorization_credential_information' }
+      it 'creates a new MockCredentialInfo' do
```

Comment:
You won't need to mock this anymore, since `MockCredentialInfo` is an object that exists. You'll be able to create the object before the spec and then test that the function retrieves it.